src/fare-contracts/components/ConsumeCarnetButton.tsx

```diff
@@ -0,0 +1,40 @@
+import {TicketValid} from '@atb/assets/svg/mono-icons/ticketing';
```

Comment:
Should this file be called `ConsumeCarnetSectionItem.tsx`?

Reply:
Yes, good catch! Fixed this and your other comment in 9665b1ce6fbdeba4e4aae8ba056467e3241ca58f